3MI0800270/2023-03-07/zad3.cpp

```diff
@@ -0,0 +1,84 @@
+#include <fstream>
+#include <iostream>
+using namespace std;
+
+struct Pair {
+    int a;
+    int b;
+};
+
+struct Relation {
+    Pair pairs[25];
+    int count = 0;
+};
+
+Pair createPair(int x, int y) {
+    return Pair{ x, y };
+}
+
+void initPair(Pair& toInit, int x, int y) {
+    toInit = { x, y };
+}
+
+Pair readPairFromFile(std::ifstream& inFile) {
+    int a, b;
+    inFile >> a >> b;
+    return Pair{ a, b };
+}
+
+void writePairToFile(std::ofstream& outFile, const Pair& toWrite) {
+    outFile << toWrite.a << " " << toWrite.b;
+}
+
+bool addPair(Relation& arg, const Pair& toInsert) {
+    if (arg.count >= 25) {
+        cout << "Error";
+        return 0;
```

Comment:
It's so much nicer to use `true` & `false` instead of `0` & `1` for boolean value representations :))